src/midi/midi_fluidsynth.cpp

```diff
@@ -44,6 +43,191 @@
+int  	(*fluid_synth_set_chorus_group_nr)(fluid_synth_t *synth, int fx_group, int nr) 			= nullptr;
+int  (*fluid_synth_set_chorus_group_level)(fluid_synth_t *synth, int fx_group, double level) 	= nullptr;
+int  (*fluid_synth_set_chorus_group_speed)(fluid_synth_t *synth, int fx_group, double speed)	= nullptr;
+int  (*fluid_synth_set_chorus_group_depth)(fluid_synth_t *synth, int fx_group, double depth_ms) = nullptr;
+int   (*fluid_synth_set_chorus_group_type)(fluid_synth_t *synth, int fx_group, int type) 		= nullptr;
+
+int  				 (*fluid_synth_reverb_on)(fluid_synth_t *synth, int fx_group, int on) 			= nullptr;
+int  (*fluid_synth_set_reverb_group_roomsize)(fluid_synth_t *synth, int fx_group, double roomsize) 	= nullptr;
+int      (*fluid_synth_set_reverb_group_damp)(fluid_synth_t *synth, int fx_group, double damping) 	= nullptr;
+int     (*fluid_synth_set_reverb_group_width)(fluid_synth_t *synth, int fx_group, double width) 	= nullptr;
+int     (*fluid_synth_set_reverb_group_level)(fluid_synth_t *synth, int fx_group, double level) 	= nullptr;
+
+int  		   (*fluid_synth_sfcount)(fluid_synth_t *synth)                                          = nullptr;
+int  			(*fluid_synth_sfload)(fluid_synth_t *synth, const char *filename, int reset_presets) = nullptr;
+void 		  (*fluid_synth_set_gain)(fluid_synth_t *synth, float gain)                              = nullptr;
+int  (*fluid_synth_set_interp_method)(fluid_synth_t *synth, int chan, int interp_method)             = nullptr;
+int  		   (*fluid_synth_noteoff)(fluid_synth_t *synth, int chan, int key)                       = nullptr;
+int  			(*fluid_synth_noteon)(fluid_synth_t *synth, int chan, int key, int vel)              = nullptr;
+int  	  (*fluid_synth_key_pressure)(fluid_synth_t *synth, int chan, int key, int val)              = nullptr;
+int                 (*fluid_synth_cc)(fluid_synth_t *synth, int chan, int ctrl, int val)             = nullptr;
+int     (*fluid_synth_program_change)(fluid_synth_t *synth, int chan, int program)                   = nullptr;
+int   (*fluid_synth_channel_pressure)(fluid_synth_t *synth, int chan, int val)                       = nullptr;
+int         (*fluid_synth_pitch_bend)(fluid_synth_t *synth, int chan, int val)                       = nullptr;
+
+int        (*fluid_synth_sysex)(fluid_synth_t *synth, 
+                                const char *data, 
+                                int len,
+                                char *response, 
+                                int *response_len, 
+                                int *handled, 
+                                int dryrun) = nullptr;
+int  (*fluid_synth_write_float)(fluid_synth_t *synth, 
+                                int len,
+								void *lout, 
+                                int loff, 
+                                int lincr,
+								void *rout, 
+                                int roff, 
+                                int rincr) = nullptr;
```

Comment:
Yeah, all that formatting looks good in Visual Studio, but it's all over the place here because you used a mixture of tabs and spaces. Can you replace all tabs with spaces in this block with your editor? Then it's perfect 😄

Reply:
Oh FFS. 

I think it's partly because I'm editing in VS code and compiling in VS 2022.